.evergreen/scripts/compile-test-azurekms.sh

```diff
@@ -17,7 +17,10 @@ git clone --depth=1 https://github.com/mongodb/libmongocrypt --branch 1.7.0
     git checkout -q 4c4aa8bf # Allows -DENABLE_MONGOC=OFF.
```

Comment:
Suggest updating the commit after https://github.com/mongodb/libmongocrypt/pull/571 is merged.

Reply:
Commit updated as suggested; piping of xtrace is no longer necessary and thus removed.